lib/SPIRVProducerPass.cpp

```diff
@@ -871,9 +871,11 @@ void SPIRVProducerPass::outputHeader() {
   case SPIRVVersion::SPIRV_1_6:
     minor = 6;
     break;
+  #if !defined(__clang__)
   default:
     llvm_unreachable("unhandled spir-v version");
     break;
+  #endif
```

Comment:
For Clang / LLVM, these probably don't make sense anymore as they are covered, if the warnings are kept as errors.